gammapy/datasets/map.py

```diff
@@ -256,15 +256,16 @@ def data_shape(self):
         """Shape of the counts or background data (tuple)"""
         return self._geom.data_shape
 
-    def npred(self):
+    def npred(self, use_cache=True):
```

Comment:
Maybe move `use_cache` to `MapDataset.__init__()` so that it becomes really configurable by the user.